src/status_im/ui/components/list_item/views.cljs

```diff
@@ -18,7 +18,9 @@
       ;;Image
       (when image
         [react/view {:margin-left 16}
-         [image]])
+         (if (vector? image)
```

Comment:
What's the use case for this?

Reply:
image can be provided as  a render function or as a vector , for example `:image render` or `:image [render]`